rust-toolchain

```diff
@@ -1 +1 @@
-1.36.0
\ No newline at end of file
+nightly
```

Comment:
Nit: just add a new line at the end of the file.

Reply:
Done!